CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ App Center Data is [retired](https://aka.ms/MBaaS-retirement-blog-post) and has
 
 * **[Fix]** Allow whitespace characters between secrets like in versions before `2.6.0`.
 
+#### WPF
+
+* **[Fix]** Fix application crash when the `App Center` configuration file is corrupted.
```

Comment:
```suggestion
* **[Fix]** Fix application crash when the `AppCenter` configuration file is corrupted.
```

Reply:
It's either App Center (with space as a service), or AppCenter when implying SDK module. Which one is it here?

Reply:
It's the name of the file.